scripts/dev.js

```diff
@@ -10,8 +10,9 @@ tasks.copyAssets('dev');
 
 console.log('[Webpack Dev]');
 console.log('-'.repeat(80));
-console.log('If you\'re developing Inject page,');
+console.log('In order to allow access to inject.js');
 console.log('please allow `https://localhost:3000` connections in Google Chrome,');
+console.log('which can be done by enabling #allow-insecure-localhost at chrome://flags');
```

Comment:
Am I right here? I'm literally not sure. But as far as I experienced when trying to make dev run, it was no matter what you were developing, in order to make the Chrome extension run, you needed to get access to https://localhost:3000. And the intended way to do it is the way I described it yeah? That's what I found through googling.

Also, would it maybe be possible to just request localhost over http instead of https? Or is that part of the webpack-server module?